wordpress-mobile/WordPress-Android
# Improvements in Feature Config

## changes
The changes in this PR is described in this [internal p2](pcdrpt-2z2) 

## To test:
1. Verify that the debug settings can be accessed from the places described in the P2
2. Verify that the debug settings can't be accesed in a release build 
3. verify that the functionality in debug settings page is as expected

## Regression Notes
1. Potential unintended areas of impact
debug settings are not working as expected

2. What I did to test those areas of impact (or what existing automated tests I relied on)
manual testing 

3. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:

UI Changes testing checklist:
